project_projected_hours/views/project_task.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+
+        <record model="ir.ui.view" id="view_task_form2_inherited">
+            <field name="name">project.task.form.inherited</field>
+            <field name="model">project.task</field>
+            <field name="inherit_id" ref="hr_timesheet.view_task_form2_inherited" />
+            <field name="groups_id" eval="[(6,0, (ref('hr_timesheet.group_hr_timesheet_user'),))]"/>
+            <field name="arch" type="xml">
+                <field name="remaining_hours" position="after">
+                    <field name="projected_hours" widget="float_time" class="oe_subtotal_footer_separator"/>
```

Comment:
Hello @rivo2302 

The display of projected hours is off-set because the native form view of task has changed structure.
It needs to put the widget "timesheet_uom" in place of "time_float" and declare the label in the class of the label "o_td_label", then add the field "projected_hours" without a label.
See code [here](https://github.com/odoo/odoo/blob/16.0/addons/hr_timesheet/views/project_views.xml#L236)